docs/install/helm.md

```diff
@@ -94,7 +94,7 @@ kubectl apply -f <output_dir>/ako/crds/
 helm list -n avi-system
 
 NAME          	NAMESPACE 	REVISION	UPDATED                             	    STATUS  	CHART    	APP VERSION
-ako-1593523840	avi-system	1       	2020-09-16 13:44:31.609195757 +0000 UTC	    deployed	ako-1.6.3	1.6.3
+ako-1593523840	avi-system	1       	2020-09-16 13:44:31.609195757 +0000 UTC	    deployed	ako-1.6.4	1.6.4
```

Comment:
The list command will give the older version say 1.6.3 I guess.

Reply:
Addressed